tools/deployment/k8s/s3/deploy.sh

```
@@ -0,0 +1,91 @@
+        }
+    ]
+}
+EOP
+)
+aws s3api put-bucket-policy --endpoint-url "${AWS_ENDPOINT_URL}" --bucket "${LOG_VIEWER_BUCKET}" --policy "$POLICY"
+
+# Upload all assets to object store at the root of the provided bucket
+# Note that uploads can fail with invalid/unknown checksum sent error. This is a known issue and not fatal.
+# See this GitHub issue for details: https://github.com/minio/minio/pull/19680
+echo "Uploading YScope log-viewer assets to object store"
+cd ${DECOMPRESSED_ASSETS_DIRECTORY}
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)